### DISCUSSION TITLE
blog/2023-05-07-causal-inference-with-count-regression/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#15] [Comments]
# blog/2023-05-07-causal-inference-with-count-regression/

Part 5 of the GLM and causal inference series.

https://solomonkurz.netlify.app/blog/2023-05-07-causal-inference-with-count-regression/

**Comment by mohelm** [comment]
Hi Solomon - thanks for the very interesting article. Although it is not the focus of you article, I wanted to ask you how you would interpret the (frequentist) results? It seems like that all of the confidence bands presented do contain zero. I am not so much interested in an interpretation regarding the concrete application from which you took the data, but more in a "generic" one for this type of result (if that is possible). Thanks!

**Reply by ASKurz** [reply]
Whether frequentist or Bayesian, I would interpret the results from the perspective of effect sizes. That is, I'd report the ATE and its 95% interval (computed however...) and discuss how impressed I was by its magnitude. In the case of this study, about 1 fewer seizures every two weeks doesn't seem all that great, to me. Sure, it's better than nothing, but that's a pretty low bar for those who are having seizures on a regular basis.

**Reply by mohelm** [reply]
OK, and in this case - not so impressive effect sizes - i suppose the c.i.s do not matter that much?

And then, if you were impressed by the magnitude of the effects, how would you comment on the c.i.s?

Thanks!

**Reply by ASKurz** [reply]
I may have miscommunicated. The uncertainty of the effect size matters regardless of its magnitude, and I think it’s a good idea to talk about it in, say, a Discussion section or during a symposium presentation. In my comment, above, I was just trying to make my point with brevity.
